sources/packages/backend/libs/sims-db/src/entities/student.model.ts

```diff
@@ -82,4 +84,21 @@ export class Student extends RecordDataModel {
     name: "sin_consent",
   })
   sinConsent: boolean;
+  /**
+   * List of all student restriction ids.
+   */
+  @RelationId((student: Student) => student.studentRestrictions)
+  studentRestrictionsIds: number[];
+  /**
+   * List of all student restrictions.
+   */
+  @OneToMany(
+    () => StudentRestriction,
+    (studentRestriction) => studentRestriction.student,
+    {
+      eager: false,
+      cascade: true,
```

Comment:
Since this is new I believe that we can have the cascade as `false` or `["update"]`. Does it make sense?